src/main/java/Car.java

```diff
@@ -70,7 +70,7 @@ public void newLocation(double x, double y, double distanceIn) {
      */
     @Override
     public String toString() {
-        return "Car #:" + id + "\tTime traveled: " + time;
+        return (id + 1) + "\t\t" + String.format("%.1f", time);
```

Comment:
String.format is very similar to how printf formats strings.
The big difference is that String.format returns a String whereas printf returns a void.
In this case, %.1f , means to take the double called time, and round it to the 1 decimal place.